pkgs/by-name/re/renode/package.nix

```
@@ -0,0 +1,109 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, writeScript
+
+, glibcLocales
+, python3Packages
+
+, gtk-sharp-2_0
+, gtk2-x11
+, screen
+
+, unstableVersion ? false
```

A better name:

```suggestion
, buildUnstable ? false
```

Or even better, 

```suggestion
, buildVariant ? "stable" # can be "unstable" or "stable"
```